page.html

```diff
@@ -0,0 +1,195 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
- [ ] Please change the name of this file to `index.html` because it is the default file name that gets opened by all browsers. By doing so visitors can visit the website without the need to write `/page.html` at the end of the URL.

Reply:
**[REQURED CHANGES]**

*Note: I will leave a general review, kindy address **ALL THE CHANGES REQUIRED BY THE PREVIOUS REVIEWER**.*

:point_right: *You've made a submission from your `animations-transitions` branch instead of the `desktop-version` branch* and the code reviewer has reviewed the `desktop-version` branch from his end and didn't disclose this issue to you. As of right now, you've addressed a few of the changes he proposed on the `animations-transitions` branch.
- As of now, your `desktop-version` is not close to the Figma template you used. 
   - **Please, kindly make all the required changes he proposed in the `desktop-version` branch.** 
   - After you can submit for a re-review and the next reviewer would perform the review upon ***the previous reviewer required changes***

:point_right: *Please, I'm sorry for any inconveniences this might cause you. As this would help reduce the confusion the review is having now*. Also, always ensure you're working from the correct branch.. :pray: